bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -70,13 +70,13 @@ object SpProgramFactory {
     prog.setThesis(isThesis(proposal))
 
     prog.setPIInfo(piInfo(proposal))
-    hostNgoEmail(proposal) foreach { e => prog.setPrimaryContactEmail(e) }
+    hostNgoEmail(proposal) foreach prog.setPrimaryContactEmail
```

Comment:
On a trivial note, we've never really documented Scala standards and we're a bit all over the place, but for some reason I'm inclined to prefer `hostNgoEmail(proposal).foreach(prog.setPrimaryContactEmail)` in this case.  Seems to be the ["official" recommendation](http://docs.scala-lang.org/style/method-invocation) too.

👍 for the eta-reduction, btw!